# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../eject/no-i18n.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

uClibC and dietlibc without add-on libraries do not have i18n.

  - Rene Rebe <rene@exactcode.de>

--- eject-2.1.0/i18n.h.vanilla	2007-01-15 10:20:57.000000000 +0000
+++ eject-2.1.0/i18n.h	2007-01-15 10:25:54.000000000 +0000
@@ -7,8 +7,15 @@
     程式國際化設計:  謝崑中
 */
 
+#include <stdlib.h> /* uClibC / dietlibc defines */
+
 #ifndef __i18n__
    #define __i18n__
+#if defined (__UCLIBC__) || defined (__dietlibc__)
+   #define _(str) (str)
+   #define N_(str) (str)
+   #define I18NCODE 
+#else
    #define PKG "eject"
    #define LOCALEDIR "/usr/share/locale"
 
@@ -20,4 +27,5 @@
 
    void i18n_init (void);
 #endif
+#endif
 
